api/Window.json

```diff
@@ -2386,40 +2403,103 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": "3.0"
+              "version_added": "3.0",
+              "partial_implementation": true,
+              "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
             },
             "webview_android": {
-              "version_added": "37"
+              "version_added": "37",
+              "partial_implementation": true,
+              "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "secure_context_required": {
```

Comment:
I don't think we should have a subentry like this for events. The one in https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API#browser_compatibility is enough.